pkg/metrics/errormetrics/errormetrics.go

```diff
@@ -84,10 +85,17 @@ var (
 	}, []string{"opcode", "error_type"})
 )
 
-func InitMetrics(registry *prometheus.Registry) {
-	registry.MustRegister(ErrorTotal)
-	registry.MustRegister(HandlerErrors)
+func RegisterMetrics(group metrics.Group) {
+	group.MustRegister(ErrorTotal)
+	group.MustRegister(HandlerErrors)
+}
 
+// TODO:
+//  1. Define metrics using functions from pkg/metrics
+//  2. Move initialization code to metrics definitions if needed or remove it
+//     if not needed
+//  3. Use label values defined as metrics.ConstrainedLabel
```

Comment:
Do these still make sense? and if so maybe put them in issues?

Reply:
Yes, definitely, I'll create issues :+1: